### ISSUE TITLE
PHP Warning appears parameter is an array

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/minify#261]
```
Are you sure this is not a problem with your configuration? (ask on the
Google Group)

Minify commit/version: All (currently using 2.1.4 )
PHP version: All

What steps will reproduce the problem?
1. Builder is enabled
2. Open https://fluidsignal.timedoctor.com/v2/views/min/b=js&f%5B%5D=bla

Explanation: preg fails when the f parameter is an array, and PHP errors out 
(depending on the server configuration). This error includes server paths.

Solution:
Verify the parameter is not an array
http://php.net/manual/en/function.is-array.php

Expected output:
Nothing.

Actual output:
Warning: preg_match() expects parameter 2 to be string, array given in 
/var/www/cysoco/v2/views/min/lib/Minify/Controller/MinApp.php on line 92 


```

---

Original issue reported on code.google.com by `rfmu...@fluidsignal.com` on 29 May 2012 at 10:35

Attachments:
- [2012-05-29-173223_1366x768_scrot.png](https://storage.googleapis.com/google-code-attachments/minify/issue-261/comment-0/2012-05-29-173223_1366x768_scrot.png)

**Comment by GoogleCodeExporter** [comment]
```
Good catch.
```

---

Original comment by `mrclay....@gmail.com` on 30 May 2012 at 12:00
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Hello, is there any temporal patch for this? I would like to point it out to 
somebody using Minify. Thanks
```

---

Original comment by `rfmu...@fluidsignal.com` on 4 Jun 2012 at 3:34

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 666cd6d89a22:
For manual patching (I don't recomment) see: 
http://code.google.com/p/minify/source/diff?spec=svn666cd6d89a224bbf3715b07be5f2
430c4c226a03&old=f7d73dcc566c99223138b525b37f6c6465a6704b&r=666cd6d89a224bbf3715
b07be5f2430c4c226a03&format=unidiff&path=%2Fmin%2Flib%2FMinify%2FController%2FMi
nApp.php
```

---

Original comment by `mrclay....@gmail.com` on 4 Jun 2012 at 3:52
- Changed state: **Verified**
- Added labels: **Release-2.1.5**
